tests/e2e/RUN-WITH-SOURCE-synpress-env-KovanOE.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Is there a reason we need a whole bash script instead of loading the ENVs from `.env`?